scripts/background.js

```diff
@@ -0,0 +1,13 @@
+chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
+  const tabUrl = tab.url ?? tab.pendingUrl;
+  if (
+    changeInfo.status === "complete" &&
+    tabUrl &&
+    tabUrl.includes("linkedin.com")
```

Comment:
Only injects the custom CSS if the URL has `linkedin.com` in it. This could apply to other sites technically but the extension itself doesn't try to do anything outside of LinkedIn due to the host permissions.